apps/src/code-studio/dancePartySongArtistTags.js

```diff
@@ -82,5 +82,6 @@ export const SongTitlesToArtistTwitterHandle = {
   showdaspoderosas_anitta: 'Anitta',
   janti_muratboz: 'MuratBoz',
   gee_girlsgen: 'GirlsGeneration',
-  growingpains_alessiacara: 'AlessiaCara'
+  growingpains_alessiacara: 'AlessiaCara',
+  despedidaycierre_vanesamartin: 'vanesamartin_'
```

Comment:
it looks like there's an ordering we should follow in this file -- this section has a comment that says

```javascript
// These tracks are currently only potential. If we get licensing, we'll be
// able to tweet the artist without an additional DTP.
```

we should probably add this to the "2021 Songs" section

Reply:
Ahh thanks - updated!